src/lib/app/types/layout.ts

```diff
@@ -15,3 +15,10 @@ export type ActionButton = {
     disabled?: boolean
     loading?: boolean
 }
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type FilterCheckbox = {
+    label: string
+    onChange: (..._: any[]) => void
+    value?: string
```

Comment:
you can remove this optional `?`

Reply:
With the changes I applied the value can still be undefined.

Reply:
sorry I didnt see it is a string but this value must be a boolean